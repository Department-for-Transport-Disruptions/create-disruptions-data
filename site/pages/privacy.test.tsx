import { render } from "@testing-library/react";
import { describe, expect, it } from "vitest";
import Privacy from "./privacy.page";

describe("privacy", () => {
    it("should render correctly", () => {
        const { container } = render(<Privacy />);
        expect(container).toMatchSnapshot();
    });
});
