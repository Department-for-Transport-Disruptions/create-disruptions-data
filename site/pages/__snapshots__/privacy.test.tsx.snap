// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`privacy > should render correctly 1`] = `
<div>
  <div
    class="app-phase-banner__wrapper"
  >
    <div
      class="govuk-phase-banner max-w-[960px] w-[90%] border-0 mx-auto app-phase-banner app-width-container"
    >
      <p
        class="govuk-phase-banner__content"
      >
        <strong
          class="govuk-tag govuk-phase-banner__content__tag"
        >
          Beta
        </strong>
        <span
          class="govuk-phase-banner__text"
        >
          This is a new service – your
           
          <a
            class="govuk-link"
            href="/feedback"
            id="feedback-link"
          >
            feedback
          </a>
           
          will help us to improve it.
        </span>
      </p>
    </div>
  </div>
  <div
    class="govuk-width-container"
  >
    <button
      class="govuk-back-link mb-7"
      type="button"
    >
      Back
    </button>
    <main
      class="govuk-main-wrapper"
      id="main-content"
    >
      <h1
        class="govuk-heading-l"
      >
        Privacy policy for the Create Transport Disruption Data Service
      </h1>
      <p
        class="govuk-body"
      >
        Last updated: 12/08/2024
      </p>
      <h2
        class="govuk-heading-m"
      >
        Who we are
      </h2>
      <p
        class="govuk-body"
      >
        The Create Transport Disruption Data Service allows users to convert their disruptions information into a format called SIRI-SX, which is an XML-based data standard. The service provides an easy-to-use webflow that takes the user through a series of questions, allowing them to define their disruptions. In order to make the process as simple as possible, we utilise data from Traveline National Dataset (TNDS), NaPTAN dataset and National Operator Code (NOC) database to represent bus operator routes and services to the user. The aim of the tool is to integrate closely with the Bus Open Data Service (BODS) in the publication of disruptions data.
      </p>
      <h2
        class="govuk-heading-m"
      >
        What data we collect from you
      </h2>
      <p
        class="govuk-body"
      >
        Our main purpose is ensuring data about local bus services across England is in the public domain. This data is provided by bus operators and anybody can sign up to use it. The personal data we collect from you includes:
      </p>
      <ol
        class="govuk-list govuk-list--bullet"
      >
        <li>
          your questions, queries or feedback, and email address if you contact GOV.UK
        </li>
        <li>
          your email address and subscription preferences when you sign up to our email alerts
        </li>
        <li>
          your Internet Protocol (IP) address, and web browser version
        </li>
        <li>
          information on how you use the site, using cookies and page tagging
        </li>
      </ol>
      <h2
        class="govuk-heading-m"
      >
        Why we need your data
      </h2>
      <p
        class="govuk-body"
      >
        We want data consumers to be able to use disruptions for bus services across England, for the benefit of the general public. We collect information through Google Analytics to see how you use this service so that we can make it better. We also collect data in order to:
      </p>
      <ol
        class="govuk-list govuk-list--bullet"
      >
        <li>
          reply to any feedback you send us, if you’ve asked us to
        </li>
        <li>
          monitor use of the site to identify security threats
        </li>
      </ol>
      <h2
        class="govuk-heading-m"
      >
        What we do with your data
      </h2>
      <p
        class="govuk-body"
      >
        We use your data to occasionally check your activity when using the service. We will not:
      </p>
      <ol
        class="govuk-list govuk-list--bullet"
      >
        <li>
          sell or rent your data to third parties
        </li>
        <li>
          share your data with third parties for marketing purposes
        </li>
      </ol>
      <p
        class="govuk-body"
      >
        We will share your data if we are required to do so by law – for example, by court order, or to prevent fraud or other crime. We use your data to calculate performance metrics to support continuous improvement of the service. When you sign up, you can also opt in to be contacted to participate in user research. If you change your mind later about this, please let us know by emailing us.
      </p>
      <p
        class="govuk-body"
      >
        We use Google Analytics software to collect information about how you use GOV.UK. This includes IP addresses. The data is anonymised before being used for analytics processing. Google Analytics processes anonymised information about:
      </p>
      <ol
        class="govuk-list govuk-list--bullet"
      >
        <li>
          the pages you visit on GOV.UK
        </li>
        <li>
          how long you spend on each GOV.UK page
        </li>
        <li>
          how you got to the site
        </li>
        <li>
          what you click on while you’re visiting the site
        </li>
      </ol>
      <p
        class="govuk-body"
      >
        We do not store your personal information through Google Analytics (for example your name or address). We will not identify you through analytics information, and we will not combine analytics information with other data sets in a way that would identify who you are. We continuously test and monitor our data protection controls to make sure they’re effective and to detect any weaknesses.
      </p>
      <h2
        class="govuk-heading-m"
      >
        How long we keep your data for
      </h2>
      <p
        class="govuk-body"
      >
        We will retain your data for as long as you have a Create Transport Disruption account. However, if you use an institutional email account and depending on what you choose for your account name, this may not include any personally identifiable information.
      </p>
      <h2
        class="govuk-heading-m"
      >
        Where your data is processed and stored
      </h2>
      <p
        class="govuk-body"
      >
        We design, build and run our systems to make sure that your data is as safe as possible at any stage, both while it’s processed and when it’s stored. Your personal data will not be transferred outside of the European Economic Area (EEA).
      </p>
      <h2
        class="govuk-heading-m"
      >
        How we protect your data and keep it secure
      </h2>
      <p
        class="govuk-body"
      >
        We are committed to doing all that we can to keep your data secure. To prevent unauthorised access or disclosure we have put in place technical and organisational procedures to secure the data we collect about you – for example, we protect your data using varying levels of encryption. We also make sure that any third parties that we deal with have an obligation to keep all personal data they process on our behalf secure.
      </p>
      <h2
        class="govuk-heading-m"
      >
        Children’s privacy protection
      </h2>
      <p
        class="govuk-body"
      >
        We understand the importance of protecting children’s privacy online. Our services are not designed for, or intentionally targeted at, children 13 years of age or younger. It is not our policy to intentionally collect or maintain data about anyone under the age of 13.
      </p>
      <h2
        class="govuk-heading-m"
      >
        What are your rights
      </h2>
      <p
        class="govuk-body"
      >
        You have the right to request:
      </p>
      <ol
        class="govuk-list govuk-list--bullet"
      >
        <li>
          information about how your personal data is processed
        </li>
        <li>
          a copy of that personal data - this copy will be provided in a structured, commonly used and machine-readable format
        </li>
        <li>
          that anything inaccurate in your personal data is corrected immediately
        </li>
      </ol>
      <p
        class="govuk-body"
      >
        You can also:
      </p>
      <ol
        class="govuk-list govuk-list--bullet"
      >
        <li>
          raise an objection about how your personal data is processed
        </li>
        <li>
          request that your personal data is erased if there is no longer justification for it
        </li>
        <li>
          ask that the processing of your personal data is restricted in certain circumstances
        </li>
      </ol>
      <p
        class="govuk-body"
      >
        If you have any of these requests, get in contact with our Data Protection Officer - you can find their contact details below.
      </p>
      <h2
        class="govuk-heading-m"
      >
        Changes to this notice
      </h2>
      <p
        class="govuk-body"
      >
        We may modify or amend this privacy notice at our discretion at any time. When we make changes to this notice, we will amend the last modified date at the top of this page. Any modification or amendment to this privacy notice will be applied to you and your data as of that revision date. We encourage you to periodically review this privacy notice to be informed about how we are protecting your data.
      </p>
      <h2
        class="govuk-heading-m"
      >
        Contact us or make a complaint
      </h2>
      <p
        class="govuk-body"
      >
        Contact the GDS Privacy Team if you either:
      </p>
      <ol
        class="govuk-list govuk-list--bullet"
      >
        <li>
          have any questions about anything in this document
        </li>
        <li>
          think your personal data has been misused or mishandled
        </li>
      </ol>
      <p
        class="govuk-body"
      >
        GDS Privacy Team
      </p>
      <p
        class="govuk-body"
      >
        <a
          class="govuk-link"
          href="mailto:gds-privacy-office@digital.cabinet-office.gov.uk"
        >
          gds-privacy-office@digital.cabinet-office.gov.uk
        </a>
         
      </p>
      <p
        class="govuk-body"
      >
        You can also contact the Cabinet Office Data Protection Officer by email or post.
      </p>
      <div>
        <p
          class="govuk-body govuk-!-margin-bottom-0"
        >
          Data Protection Officer
        </p>
        <p
          class="govuk-body"
        >
          <a
            class="govuk-link"
            href="mailto:DPO@cabinetoffice.gov.uk"
          >
            DPO@cabinetoffice.gov.uk
          </a>
        </p>
      </div>
      <div>
        <p
          class="govuk-body govuk-!-margin-bottom-0"
        >
          Data Protection Officer
        </p>
        <p
          class="govuk-body govuk-!-margin-bottom-0"
        >
          Cabinet Office
        </p>
        <p
          class="govuk-body govuk-!-margin-bottom-0"
        >
          70 Whitehall
        </p>
        <p
          class="govuk-body"
        >
          London SW1A 2AS
        </p>
      </div>
      <p
        class="govuk-body"
      >
        If you have a complaint, you can also contact the Information Commissioner's Office (ICO). The ICO is an independent regulator set up to uphold information rights.
      </p>
    </main>
    <div>
      <h2
        class="govuk-heading-s"
      >
        Help and Support
      </h2>
      <p
        class="govuk-body"
      >
        If you are having problems, please contact the Create Disruption Service via this link:
         
        <a
          class="govuk-link govuk-!-font-size-19"
          href="/contact"
        >
          Contact us
        </a>
      </p>
    </div>
  </div>
  <footer
    class="govuk-footer"
    role="contentinfo"
  >
    <div
      class="govuk-width-container"
    >
      <div
        class="govuk-footer__meta"
      >
        <div
          class="govuk-footer__meta-item govuk-footer__meta-item--grow"
        >
          <h2
            class="govuk-visually-hidden"
          >
            Support links
          </h2>
          <ul
            class="govuk-footer__inline-list"
          >
            <li
              class="govuk-footer__inline-list-item"
            >
              <a
                class="govuk-footer__link"
                href="/contact"
              >
                Help
              </a>
            </li>
            <li
              class="govuk-footer__inline-list-item"
            >
              <a
                class="govuk-footer__link"
                href="/cookies"
              >
                Cookies
              </a>
            </li>
            <li
              class="govuk-footer__inline-list-item"
            >
              <a
                class="govuk-footer__link"
                href="/contact"
              >
                Contact
              </a>
            </li>
            <li
              class="govuk-footer__inline-list-item"
            >
              <a
                class="govuk-footer__link"
                href="/accessibility"
              >
                Accessibility
              </a>
            </li>
            <li
              class="govuk-footer__inline-list-item"
            >
              <a
                class="govuk-footer__link"
                href="https://www.gov.uk/help/terms-conditions"
              >
                Terms and conditions
              </a>
            </li>
            <li
              class="govuk-footer__inline-list-item"
            >
              <a
                class="govuk-footer__link"
                href="/privacy"
              >
                Privacy
              </a>
            </li>
          </ul>
          <p
            class="govuk-footer__body"
          >
            Built by the
             
            <a
              class="govuk-footer__link"
              href="https://www.gov.uk/government/organisations/department-for-transport"
              rel="noreferrer"
              target="_blank"
            >
              Department for Transport
            </a>
          </p>
          <svg
            aria-hidden="true"
            class="govuk-footer__licence-logo"
            focusable="false"
            height="17"
            viewBox="0 0 483.2 195.7"
            width="41"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M421.5 142.8V.1l-50.7 32.3v161.1h112.4v-50.7zm-122.3-9.6A47.12 47.12 0 0 1 221 97.8c0-26 21.1-47.1 47.1-47.1 16.7 0 31.4 8.7 39.7 21.8l42.7-27.2A97.63 97.63 0 0 0 268.1 0c-36.5 0-68.3 20.1-85.1 49.7A98 98 0 0 0 97.8 0C43.9 0 0 43.9 0 97.8s43.9 97.8 97.8 97.8c36.5 0 68.3-20.1 85.1-49.7a97.76 97.76 0 0 0 149.6 25.4l19.4 22.2h3v-87.8h-80l24.3 27.5zM97.8 145c-26 0-47.1-21.1-47.1-47.1s21.1-47.1 47.1-47.1 47.2 21 47.2 47S123.8 145 97.8 145"
              fill="currentColor"
            />
          </svg>
          <span
            class="govuk-footer__licence-description"
          >
            All content is available under the
             
            <a
              class="govuk-footer__link"
              href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
              rel="license"
            >
              Open Government Licence v3.0
            </a>
            , except where otherwise stated
          </span>
        </div>
        <div
          class="govuk-footer__meta-item"
        >
          <a
            class="govuk-footer__link govuk-footer__copyright-logo bg-govuk-crest"
            href="https://www.nationalarchives.gov.uk/information-management/re-using-public-sector-information/uk-government-licensing-framework/crown-copyright/"
          >
            © Crown copyright
          </a>
        </div>
      </div>
    </div>
  </footer>
</div>
`;
